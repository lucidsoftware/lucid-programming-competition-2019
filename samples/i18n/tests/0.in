5
i
am
the
cool
coder
