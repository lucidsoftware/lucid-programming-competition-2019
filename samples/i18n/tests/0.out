i
am
t1e
c2l
c3r
